From 677d22bb6f46ed7459a171efae92ee8fe2427836 Mon Sep 17 00:00:00 2001
From: Javier Pena <jpena@redhat.com>
Date: Wed, 28 Jun 2017 12:44:23 +0200
Subject: [PATCH 2/2] Remove external references

The theme includes some external scripts and references, which
could be used for tracking and web browsing statistics. Since they
are not needed for the theme purposes and add privacy concerns, we
are removing them.
---
 openstackdocstheme/theme/openstackdocs/css.html | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/openstackdocstheme/theme/openstackdocs/css.html b/openstackdocstheme/theme/openstackdocs/css.html
index 369cf8b..5079a3a 100644
--- a/openstackdocstheme/theme/openstackdocs/css.html
+++ b/openstackdocstheme/theme/openstackdocs/css.html
@@ -6,14 +6,7 @@
 
 <!-- Fonts -->
 <link href="{{pathto('_static/css/font-awesome.css', 1)}}" rel="stylesheet">
-<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,400,700' rel='stylesheet' type='text/css'>
 
 <!-- Custom CSS -->
 <link href="{{pathto('_static/css/combined.css', 1)}}" rel="stylesheet">
 
-<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-<!--[if lt IE 9]>
-    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-<![endif]-->
-- 
2.9.4

